force-app/main/default/classes/AttendanceSelector.cls

```diff
@@ -29,7 +32,12 @@ public with sharing class AttendanceSelector {
         }
 
         List<ServiceParticipant__c> serviceParticipants = [
-            SELECT Id, Contact__c, Service__c, ProgramEngagement__c
+            SELECT
```

Comment:
@bethbrains ,I think we need to add an order by clause since one of the AC in the story says that the contact names should be listed alphabetically

Reply:
Nice catch, but I would suggest we do our sorting in JS vs SOQL